city_metrix/layers/building_classifier/building_classifier.py

```diff
@@ -0,0 +1,197 @@
+import xarray as xr
```

Comment:
is this file just a duplicate of the .py file?

Reply:
This is the .py file.